pkg/ccl/backupccl/restore_job.go

```diff
@@ -657,6 +657,25 @@ func shouldPreRestore(table *tabledesc.Mutable) bool {
 	return ok
 }
 
+// eligibleInProgressImport returns true if the descriptor represents a table with an in progress
+// import that started in a cluster finalized to version 22.2.
```

Comment:
nit: should say: "import that started ina cluster finalized to version 22.2 and is getting restored on finalized 22.2 cluster"

Reply:
the restoring cluster version check is superfluous as the descriptor's start time will never be above 0 in a mixed version restore. Perhaps I should instead conduct an assertion failure after checking that the import start time is greater than 0.